src/lib/components/consignments/__stories__/index.tsx

```diff
@@ -0,0 +1,45 @@
+import "./consignments.story"
+import "./typography.story"
+
+import { storiesOf } from "@storybook/react-native"
+import * as React from "react"
+import { View } from "react-native"
+
+import * as bottomAlignedButton from "./consignments-bottom-aligned.story"
+import * as search from  "./consignments-search.story"
+import * as todo from  "./consignments-todo.story"
```

Comment:
wraps all the exports into an object, then the `AutoStory` will be checked against them, so that as new modules are added - they have to have the same 3 exports. I didn't go wild on the typing below, just wanted to get enough to not cause trouble in the future.